packages/web-worker/src/parent.ts

```diff
@@ -0,0 +1,51 @@
+export function createWorker<T>(script: () => Promise<T>): () => T {
+  return function create() {
+    if (typeof Worker === 'undefined') {
+      return new Proxy(
+        {},
+        {
+          get(_target, _property) {
+            return () => {
+              throw new Error(
+                'You can’t call a method on a worker on the server.',
+              );
+            };
+          },
+        },
+      ) as any;
+    }
+
+    const workerScript = URL.createObjectURL(
+      new Blob([`importScripts(${JSON.stringify(script)})`]),
+    );
+
+    const worker = new Worker(workerScript);
```

Comment:
Do we have any ideas on how/when we'll want to `terminate` the worker?

Reply:
I think it will just be part of the API for `createWorker()`? Not positive yet, but I think it's mostly up the consumer to terminate.